mesonbuild/minstall.py

```diff
@@ -709,6 +710,13 @@ def run_install_script(self, d: InstallData, destdir: str, fullprefix: str) -> N
 
     def install_targets(self, d: InstallData, dm: DirMaker, destdir: str, fullprefix: str) -> None:
         for t in d.targets:
+            #In AIX, we archive our shared libraries. During install section we need to install the
+            #archive in which the shared library exists. Hence in the below code we change the
+            #shared library install that can have both so_version and lt_version to archive install.
+            if mesonlib.is_aix():
+                if '.so' in t.fname:
+                    aix_fname = re.sub('[.][a]([.]?([0-9]+))*([.]?([a-z]+))*','.a',t.fname.replace ('.so', '.a'))
+                    t.fname = aix_fname
```

Comment:
Similar comments.

Reply:
This is done. Kindly check the latest commit